src/main/java/Trader.java

```diff
@@ -96,8 +96,8 @@ public static void main(String[] args) {
         List<Tradable> all_items = Arrays.asList(
                 new Horse(),
                 new Horse(),
-                new Horse()
-                // TODO: Add Tradable objects here!
```

Comment:
remove todo

Reply:
I don't see the todo part in my Trader class in IntelliJ, this is weird